### DISCUSSION TITLE
SaysWho: Quote Identification, Attribution and Resolution

### DISCUSSION THREAD

**Discussion by afriedman412** [explosion/spaCy#12824] [Show & Tell]
### **[SAYSWHO](https://github.com/afriedman412/sayswho)**

I combined [SpaCy's corefencing](https://explosion.ai/blog/coref) with [Textacy's](https://github.com/chartbeat-labs/textacy) quotation finder.

1. Find quotes
2. Attribute quotes to speakers
3. Resolves speakers to singular entities across the document

Let me know what you think!

**Comment by svlandeg** [comment]
This looks great, thanks for sharing! 

If you want, you could also contribute a PR to the spaCy [universe](https://spacy.io/universe)?
